components/contribution-flow/StepCheckout.js

```diff
@@ -7,13 +7,57 @@ import { FormattedMessage } from 'react-intl';
 import useClipboard from '../../lib/hooks/useClipboard';
 
 import Container from '../Container';
-import { Box } from '../Grid';
+import { Box, Flex } from '../Grid';
+import I18nFormatters, { I18nSupportLink } from '../I18nFormatters';
+import MessageBox from '../MessageBox';
 import StyledButton from '../StyledButton';
+import StyledLink from '../StyledLink';
 import { P, Span } from '../Text';
 
 const StepCheckout = ({ stepDetails, order }) => {
   const { isCopied, copy } = useClipboard();
 
+  const renderCryptoInformation = order => {
+    const pledgeCurrency = order?.data?.thegivingblock?.pledgeCurrency;
+    const depositAddress = order?.paymentMethod?.data?.depositAddress;
+
+    return (
+      <Flex flexDirection="column" width={1} mt="24px">
+        <MessageBox type="info" fontSize="13px" mb={2}>
+          <FormattedMessage
+            id="collective.user.orderProcessing.crypto"
+            defaultMessage="<strong>Your contribution is pending.</strong> We also sent you these details to your email. Once the transaction is completed you will receive a confirmation email with the details."
+            values={I18nFormatters}
+          />
+          {` `}
+          {['BTC', 'ETH', 'BCH', 'LTC', 'ZEC', 'DOGE'].includes(pledgeCurrency) && (
+            <FormattedMessage
+              defaultMessage="You can view the status of your transaction at the Blockchain explorer: {link}"
+              values={{
+                link: (
+                  <StyledLink
+                    openInNewTab
+                    href={`https://blockchair.com/search?q=${depositAddress}`}
+                  >{`https://blockchair.com/search?q=${depositAddress}`}</StyledLink>
+                ),
+              }}
+            />
+          )}
+        </MessageBox>
+        <Flex mt="24px" justifyContent="center">
+          <P fontSize="13px" color="black.700">
+            <FormattedMessage
+              defaultMessage="You can <SupportLink>contact support</SupportLink> if something goes wrong."
+              values={{
+                SupportLink: I18nSupportLink,
+              }}
+            />
+          </P>
+        </Flex>
```

Comment:
I don't see the point for this, the interface is already cluttered by the MessageBox above.

Reply:
This is as per design; https://www.figma.com/file/ZQBMWhnGGtRWeIZknFW1eP/%5BOC-Design%5D-Production-Ready-%E2%9C%85?node-id=15015%3A205718. I think the idea is that if the QR code is not generated for some reason the user has a way of contacting support on the page itself. But I am not against removing it so I'll do so if no objections.

Reply:
I've removed the link. 👍🏽